src/sql/workbench/services/objectExplorer/test/browser/objectExplorerService.test.ts

```diff
@@ -253,6 +255,39 @@ suite('SQL Object Explorer Service tests', () => {
 			id: 'testID2'
 		});
 		conProfGroup = new ConnectionProfileGroup('testGroup', undefined, 'testGroup', undefined, undefined);
+
+		expiredConnection = new ConnectionProfile(capabilitiesService, {
+			connectionName: 'expiredName',
+			savePassword: false,
+			groupFullName: 'expiredGroup',
+			serverName: 'expiredServerName',
+			databaseName: 'expiredDatabaseName',
+			authenticationType: 'inetgrated',
+			password: 'expired',
+			userName: 'expiredUsername',
+			groupId: undefined,
+			providerName: mssqlProviderName,
+			options: { 'expiresOn': 0 },
+			saveProfile: true,
+			id: 'expiredId'
+		});
+
+		validConnection = new ConnectionProfile(capabilitiesService, {
+			connectionName: 'validName',
+			savePassword: false,
+			groupFullName: 'validGroup',
+			serverName: 'validServerName',
+			databaseName: 'validDatabaseName',
+			authenticationType: 'inetgrated',
+			password: 'valid',
```

Comment:
add these passwords generally causes CredentialScanner errors.  I think the value may need to be 'placeholder' or something to avoid the errors.